imagersite/imager_profile/views.py

```diff
@@ -1,3 +1,57 @@
 from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import get_object_or_404
+from django.contrib.auth.models import User
+from django.forms import ModelForm
+from django.views.generic.edit import UpdateView
+from .models import ImagerProfile
+
+PROFILE_TEMPLATE = 'profile.html'
+
+
+@login_required(login_url='/login')
+def Profile(request, *args, **kwargs):
+    """Authenticated User Profile."""
+    return render(request, PROFILE_TEMPLATE)
+
+
+def edit_user_profile(request):
```

Comment:
this should probably be protected wtih the login_required decorator.  It's not of any use at all (and will break) if an unauthenticated person attempts to see it.